### ISSUE TITLE
Aborting a rebase with conflicts leaves file that were subject to permission change

### ISSUE THREAD

**Issue by mcdado** [git-up/GitUp#133]
If I try to rebase a branch onto master, but after a merge conflict I decide to cancel the operation, if on the path to master there is a commit that changes file permissions (reported as Executable » Blob) that commit will not be unstaged when the rebase is aborted.

If there is need for a sample repo let me know!

**Comment by swisspol** [comment]
Yes please provide a test repo. You can zip it and attach it to this issue.

**Comment by mcdado** [comment]
I tried reproducing the conditions in a new repo, but couldn't. I could pull an [Eddy Cue move](http://daringfireball.net/thetalkshow/2016/02/12/ep-146) and send you my repo, zipped up. How can I send a link to it? My email is mcdado at gmail, if you ping me I can reply to you with the link and more information regarding reproducing the issue.

**Comment by swisspol** [comment]
Can you provide a series of screenshots instead along with the output of `git status` at the last step?

**Comment by mcdado** [comment]
Alright! I managed to reproduce the issue by checking out the individual commits into a new repo. There isn't really anything confidential or copyrighted in this source, it's just front-end templates and stuff like css and js.

[Download repo](http://cl.ly/1e2M2V2D0F3B/download/test-repo.tgz).

Steps:

1. try rebasing `fork` on top of `master`;
1. cancel the rebase because of merge conflict;
1. go to commit view, see that there are many staged files left.

**Comment by swisspol** [comment]
Thanks, I'll have a look when I get a chance.

**Comment by mcdado** [comment]
You're welcome! 👍
